src/devices/bus/nabupc/adapter.cpp

```diff
@@ -9,65 +9,246 @@
 #include "emu.h"
 #include "adapter.h"
 
+#include "client_http.hpp"
+#include "diserial.h"
 #include "emuopts.h"
+#include "hashing.h"
 #include "unzip.h"
```

Comment:
Please separate the src/emu headers from the src/lib/util headers, and sort the headers within each group.  The src/emu headers should come before the src/lib/util headers.